site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,413 @@
+
+On the lowest level, all that a CPU does is process a list of instructions. Some instructions however take longer to execute than others. The time it takes for an instruction to execute is measured in cycles. Previously, we were using a very naive implementation of assuming every instruction would take one cycle. This leads to games idling longer than they should, therefore unnecessarily slowing down the emulation.
+Now [merryhime](https://github.com/merryhime), the author of [dynarmic](https://github.com/merryhime/dynarmic), has implemented much more accurate cycle counts, significantly reducing this issue.
+
+## Multiplayer Fixes
+
+### APT: implement Set and GetWirelessRebootInfo ([#5328](https://github.com/citra-emu/citra/pull/5328)) by [B3n30](https://github.com/B3n30)
+
+This PR implements two key services relating to Download Play multiplayer on Citra. `SetWirelessRebootInfo` and `GetWirelessRebootInfo` are used to transfer information and arguments between reboots of Download Play. These are necessary so that the app is able to start another app, like in the way Download Play works via transferring a game demo from one console to another.
+
+### NWM_UDS: Implement disconnect_reason and EjectClient ([#5331](https://github.com/citra-emu/citra/pull/5331)) by [B3n30](https://github.com/B3n30)
+
+This is required to be able to disconnect from a multiplayer session correctly if the host chooses to terminate the room or force-disconnect clients. This allows for multiplayer to work correctly in certain games, such as the Monster Hunter series.
+
+### service/nwm_uds: Various improvements/corrections ([#5382](https://github.com/citra-emu/citra/pull/5382)) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+This PR is just a minor fix to multiplayer in the Monster Hunter games, specifically Monster Hunter Generations, Monster Hunter X and Monster Hunter XX. These games would suffer from crashes and disconnects when players tried to join a lobby. Now, with this fix, you can go out and hunt your favorite monsters with your friends again!
+
+### Remove citra-room dependence on core ([#5519](https://github.com/citra-emu/citra/pull/5519)) by [vitor-k](https://github.com/vitor-k)
+
+With some reorganization of the code, we were able to reduce the size of the citra-room executable by 85%, decreasing citra’s compressed size by 6MB and uncompressed size by 43MB!
+Those who host rooms with citra-room will also no longer need to have OpenGL support on their machines.
+
+### UDS: implement GetApplicationData ([#5533](https://github.com/citra-emu/citra/pull/5533)) by [mnml](https://github.com/mnml)
+
+By implementing `GetApplicationData`, an issue is resolved in which multiplayer communications would only work one way in Dragon Ball Fusions, and fixes multiplayer altogether in Final Fantasy Explorers. `GetApplicationData` is used to load appdata from the current beacon, which can then only be used whilst connected to the network. 
```

Comment:
By implementing `GetApplicationData`, an issue is resolved in which multiplayer communications would only work one way in Dragon Ball Fusions. This change also fixes multiplayer altogether in Final Fantasy Explorers.

Reply:
Can you explain a bit more what the second sentence actually means for users? Also, "whilst" -> "while"